crud-rest-service/src/app.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
+    return 'Hello World!';
+  }
```

Comment:
I think, it's wasted service better remove

Reply:
Perhaps it will be useful in future tasks )